plugins/docker/docker_

```diff
@@ -193,7 +205,11 @@ def print_containers_status(client):
     dead = []
     for container in client.all_containers:
         if container.status == 'running':
-            running.append(container)
+            state = client.api.inspect_container(container.name)['State']
+            if 'Health' in state and state['Health']['Status'] == 'unhealthy':
```

Comment:
You could simplify this (and add a bit of robustness) with `if state.get('Health', {}).get('Status') == 'unhealthy':` - but this is just a matter of taste ...